### DISCUSSION TITLE
How to change the default rem value

### DISCUSSION THREAD

**Discussion by xu-Aurora** [tailwindlabs/tailwindcss#10118] [Help]
The default is 1=0.25rem, I would like to configure it to 1=1rem, how do I configure it?

**Comment by wongjn** [comment]
[Override the default spacing scale](https://tailwindcss.com/docs/customizing-spacing#overriding-the-default-spacing-scale)

**Reply by ProPhycient5** [reply]
> [Override the default spacing scale](https://tailwindcss.com/docs/customizing-spacing#overriding-the-default-spacing-scale)

This is the expected solution. @xu-Aurora please close the discussion by marking @wongjn reply as `Answer`.

**Reply by cdx111** [reply]
This is the expected solution. @xu-Aurora please close the discussion by marking @wongjn reply as Answer.
